docs/reference/platforms/android/gradle.rst

```diff
@@ -274,13 +274,15 @@ will result in an ``AndroidManifest.xml`` declaration of::
 
     <uses-permission android:name="android.permission.HIGH_SAMPLING_RATE_SENSORS">
 
-``target_sdk_version``
+``target_os_version``
 ----------------------
 
-The SDK version that the app will target. This controls the APIs that are available for
-use in your application. This is *not* the Android version; it is the underlying SDK
-version. For example, Android 15 uses an SDK version of 35; if you wanted to specify
-Android 15 as your target SDK version, you would define ``target_sdk_version = "35"``.
+The API level that the app will target. This controls the version of Android tools that
+are used to build your app (by setting the ``compileSdkVersion`` for your app), and the
```

Comment:
The tools version is controlled by yet another setting (`buildToolsVersion`), but that shouldn't usually be customized because it's so tightly coupled to Gradle.

```suggestion
The API level that the app will target. This controls the version of the Android SDK that
is used to build your app (by setting the ``compileSdkVersion`` for your app), and the
```